hr_timesheet_sheet/tests/test_hr_timesheet_reviewer_no_email.py

```diff
@@ -184,16 +166,36 @@ def test_notify_compute_recipients_notif_disable(self):
                 ("subtype_id.name", "=", "Approved"),
             ]
         )
-        # Check recipients list
-        recipients = timesheet._notify_compute_recipients(
+        # Get recipients data
+        rdata = self.mail_thread_obj._notify_compute_recipients(
             message=message_done, msg_vals=msg_dict
-        )["partners"]
+        )
+
+        # Call the method being tested
+        timesheet._notify_record_by_email(message=message_done, recipients_data=rdata)
+        recipient_ids = message_done.notification_ids.res_partner_id.ids
+        partner_follower_ids = timesheet.message_follower_ids.partner_id.ids
+
         # create set of recipients id
-        recipient_ids = {recipient["id"] for recipient in recipients}
+        recipient_ids = {id for id in recipient_ids}
         # create set of possible reviewers id
         possible_reviewer_ids = {reviewer.id for reviewer in possible_reviewers}
+        # create set of followers
+        partner_follower_ids = {id for id in partner_follower_ids}
```

Comment:
Could you do this: `partner_follower_ids = set(partner_follower_ids)`

Reply:
fixed